core/src/main/java/com/google/errorprone/bugpatterns/TypesWithUndefinedEquality.java

```diff
@@ -24,25 +24,21 @@
 
 /** Enumerates types which have poorly-defined behaviour for equals. */
 public enum TypesWithUndefinedEquality {
```

Comment:
The following (somewhat hacky, likely Linux-only) script verifies that this PR does not introduce additional changes: 

```
#!/usr/bin/env bash

set -e -u -o pipefail

normalize() {
  local rev="${1}"

  git show "${rev}:core/src/main/java/com/google/errorprone/bugpatterns/TypesWithUndefinedEquality.java" \
    | grep -Pzo '(?<=LONG_SPARSE_ARRAY\()[^)]+' | grep -Pao '[\w.]+' | sort -u
  git show "${rev}:core/src/main/java/com/google/errorprone/bugpatterns/TypesWithUndefinedEquality.java" \
    | grep -Pzo '(?<=SPARSE_ARRAY\()[^)]+' | grep -Pao '[\w.]+' | sort -u
  git show "${rev}:core/src/main/java/com/google/errorprone/bugpatterns/TypesWithUndefinedEquality.java" \
    | grep -Po '[A-Z_]+\(' | sort -u
}

diff -u <(normalize master) <(normalize HEAD)
```